app/src/App.css

```
@@ -1,95 +1,163 @@
-@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Coming+Soon&family=Open+Sans:wght@300;400;600&display=swap');
+:root {
+  /* COLORS */
+  --white: #e9e7eb;
+  --gray: #f8f6ec;
+  --purple: #571089;
+  --semi-purple: #200b2f80;
+  --btn: #e4d6ff;
+  --btn-nav: #e4d6ffb3;
+  --btn-border: 1px solid #3D00B8;
+  --canvas-border: 1px solid #736f72;
+  /* */
+  --title-font: 'Coming Soon';
+}
+
+body {
+  background-color: var(--white);
+  background-image: url("./assets/beige-paper.png") repeat;
```

I think this might be your problem.  Looking at the `beige-paper.png` for your textured background, it should most certainly be visible and it's not. You declared the background color first, so it's overriding it. 

Here's my suggestion:

```suggestion
  background: url("./assets/beige-paper.png") repeat, #e9e7eb;
  background-blend-mode: multiply; 
```

You should layer your backgrounds like so, with your front being the texture image. Then, use the `background-blend-mode` property. It's great. `multiply` works by removing all white from an image such that only shadow, color, and texture remain. The image/colors behind it will come through and blend nicely. You could also do `overlay` instead for effects where you want a balance of light and dark to be applied. Also, you can control the blend modes of each layer, if you like. And `normal` will make a layer just be solid and opaque as it normally would be.